contracts/TwabRewards.sol

```diff
@@ -257,6 +261,14 @@ contract TwabRewards is ITwabRewards {
         require(succeeded && controllerAddress != address(0), "TwabRewards/invalid-ticket");
     }
 
+    /**
+        @notice Allow a promotion to be created or extended only by a positive number of epochs.
+        @param _numberOfEpochs Number of epochs to check
+    */
+    function _requireNumberOfEpochs(uint8 _numberOfEpochs) internal view {
+        require(_numberOfEpochs > 0, "TwabRewards/epochs-not-zero");
```

Comment:
might as well pull this up into the createPromo, once the above refactoring is made

Reply:
It should still revert if we don't actually extend the promotion when passing 0.